app/assets/stylesheets/govuk_publishing_components/lib/_print_support.scss

```diff
@@ -25,3 +25,48 @@
     }
   }
 }
+
+// Two helper classes to ensure that all printed links
+// receive consistent formatting. Most links inherit
+// styles from govuk-frontend, including the printing
+// of the href that a link contains. This makes printed
+// links more useful, but they can take up a lot of space
+// on the page in the default font size and will print
+// in blue by default. These helpers set the printed colour
+// to black, reduce the font size, and improve the layout
+// of the printed href.
+//
+// .gem-print-link
+// ----------------------------------------------------------
+// Typically this class will be applied to existing
+// elements that have the `govuk-link` class but it can
+// also be used on other link elements.
+//
+// .gem-print-links-within
+// ----------------------------------------------------------
+// A variation of the previous print style, to be used on
+// parent elements that contain links with the `govuk-link`
+// class. This is typically only required for views where
+// the `govuk-link` element is not exposed in the markup
+// of the view as it's coming from a self-contained helper.
```

Comment:
Have you got some examples of where this is needed?

Reply:
Yep, it's required in `inset-text` component which doesn't expose any links directly in the view, I added it here: https://github.com/alphagov/govuk_publishing_components/pull/4275/files#diff-666dbf87f00cea8384ab451ab812f206f7c93e02915ab4edd1bcfcf664040de9R13

The whole view looks like this:
```<%
  add_gem_component_stylesheet("inset-text")

  id ||= "inset-text-#{SecureRandom.hex(4)}"
  margin_top ||= 6
  margin_bottom ||= 6

  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new({
    margin_top: margin_top,
    margin_bottom: margin_bottom
  })

  classes = %w[gem-c-inset-text govuk-inset-text]
  classes << shared_helper.get_margin_top
  classes << shared_helper.get_margin_bottom
%>
<%= tag.div id: id, class: classes do %>
  <% if defined? text %>
    <%= text %>
  <% elsif block_given? %>
    <%= yield %>
  <% end %>
<% end %>```